src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:8080/api" method="post"
+    data-qa="big"
+    class="form">
```

Comment:
check out the point 5 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)